src/main/resources/stsjorbsmodResources/localization/eng/JorbsMod-Card-Strings.json

```diff
@@ -317,6 +317,10 @@
     "NAME": "Banish",
     "DESCRIPTION": "Enemy takes no action and no damage for !M! turns. Exhaust."
   },
+  "stsjorbsmod:DimensionDoor": {
+    "NAME": "Dimension Door",
+    "DESCRIPTION": "Escape from a non-boss combat and receive no rewards. NL If you escape, you may ignore paths when choosing the next room to travel to. NL Destroy this card."
```

Comment:
I think this is still allowing escape from boss combats as-is

Reply:
Oopsie.